src/hatti/ona/stream.cljs

```diff
@@ -0,0 +1,52 @@
+(ns hatti.ona.stream
+  (:require-macros [cljs.core.async.macros :refer [go]])
+  (:require [cljs.core.async :refer [<! put! chan timeout close!]]
+            [cljsjs.oboe]))
+
+(defn process-aggregated-data!
+  "Calls callback on aggregated data that come via messages in channel.
+   Callback accepts two arguments: data (cljs list), and completed? (bool)
+  If there is a parser in message, resume it after a timeout."
+  [agg-data-channel callback]
+  (go
+   (while true
+     (let [{:keys [data completed? ]} (<! agg-data-channel)]
+       (when data (callback data completed?))
+       ;; Timeout so browser can do some rendering before parsing again.
+       (<! (timeout 10))
```

Comment:
this is millis I imagine? is there a reason this specific value was chosen? should we put it into a `def`?

is the reason for the timeout in the completed condition the same?